fedimint-core/src/admin_client.rs

```diff
@@ -174,11 +181,16 @@ pub struct ConfigGenConnectionsRequest {
 }
 
 #[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+/// Connection information sent between peers in order to start config gen
 pub struct PeerServerParams {
+    /// TLS cert is necessary for P2P auth during DKG and  consensus
     #[serde(with = "serde_tls_cert")]
     pub cert: rustls::Certificate,
+    /// P2P is the network for running DKG and consensus
     pub p2p_url: Url,
+    /// API for secure websocket requests
     pub api_url: Url,
+    /// Name of the peer, used in TLS auth
     pub name: String,
```

Comment:
Side note: would be nice if this type validated that it was a valid DNS name. I know that in the past I've entered names with spaces or just a number like `3` and I get runtime error that invalid DNS name was entered. Might be better if this was validated up-front and enforced by type system.